pipeline-modules/deployment-service/aws/lambda-gateway.yaml

```diff
@@ -0,0 +1,152 @@
+        properties:
+          path:
+            title: Path
+            type: string
+            default: ""
+          name:
+            title: Name
+            type: string
+            default: ""
+          image:
+            title: Image
+            type: string
+            default: ""
+          verb:
+            title: Verb
+            type: string
+            default: ""
+    job_type:
+      title: Job Type
+      description: This is to deploy or undeploy application
+      type: string
+      default: deploy
+  required: ["region","bucket"]
+  internal:
+    - app_env
+    - repository
+    - tag
+    - applicationName
+    - environment
+    - project
+    - job_type
+template: |
+  ########################################################################################################################################################
+  # The template uses serverless framework to deploy function to lambdas using container images                                                          #
+  # Uses serverless-deployment-bucket plugin to keep iac code                                                                                            #
+  # Uses machine env from .env file https://www.serverless.com/framework/docs/environment-variables                                                           #
+  ########################################################################################################################################################
+  {% set workspace = '$CODEBUILD_SRC_DIR' %}   {# has to be a persistent volume #}
+  {% set pipeline_env_file = workspace|add:'/.env' %}
+
+  version: 0.2
+  phases:
+    install:
+      commands:
+        - aws configure set region {{region}}
+        - npm install -g npm
+        - npm install serverless -g
+        - npm install serverless-deployment-bucket -g
+    build: 
+      commands:
+        - |
+          echo "
+          service: {{applicationName}}
+          useDotenv: true
+          frameworkVersion: '3'
+          provider:
+            name: aws
+            stage: {{environment}}
+            ecr:
+              images: 
+              {% for func in funcs %} 
+                {{ func.name }}:
+                  uri: {{ func.image }}
+              {% endfor %}    
+            architecture: {{architecture}}
+            memorySize: {{memory}}
+            timeout: {{timeout}}
+            deploymentBucket:
+              name: {{ bucket }}
+          functions:
+          {% for func in funcs %} 
+            {{ func.name }}:
+              image:
+                name: {{ func.name }}
+              events:
+                - httpApi:
+                    path: {{ func.path }}
+                    method: {{ func.verb }}
+          {% endfor %}  
+          plugins:
+            - serverless-deployment-bucket  
+          " >> serverless.yml
+      {% if job_type == 'deploy' %}
+        - serverless deploy
+      {% else %}
+        - sls remove
```

Comment:
why is it "serverless deploy" but "sls remove"?

Reply:
both works, its an alias. i change it to be one